nerd-icons.el

```diff
@@ -139,6 +139,9 @@
     ("ds_store"       nerd-icons-faicon "nf-fa-cogs"             :face nerd-icons-silver)
     ("exe"            nerd-icons-octicon "nf-oct-file_binary"    :face nerd-icons-dsilver)
     ("msg"            nerd-icons-octicon "nf-oct-mail"           :face nerd-icons-dsilver)
+    ;; Diff
+    ("diff"           nerd-icons-devicon "nf-dev-git_compare"    :face nerd-icons-lred)
+    ("patch"          nerd-icons-devicon "nf-dev-git_compare"    :face nerd-icons-lred)
```

Comment:
Let me know if there's a better section to put these in.

Reply:
Maybe `nf-cod-diff` or `nf-oct-diff` is better? WDYT?

Reply:
I'd go with `nf-oct-file_diff`, personally. I went with `nf-dev-git_compare` to match `diff-mode`, but IMO `nf-oct-file_diff` works better in both cases.

Reply:
Yep, then please move forward with `nf-oct-file_diff`.

Reply:
I updated both `diff-mode` and the path/diff file icons (I assumed that's what you meant).

Reply:
(also happy to squash that commit into the 172db04, I'm not sure how you prefer to do reviews)